packages/discovery-provider/ddl/migrations/0109_backfill_prod_audio_analysis.sql

```diff
@@ -0,0 +1,4448 @@
+  update tracks set bpm = 129, musical_key = 'G flat major' where audio_upload_id = '01JCP7425J2DJJETWEZB0DXGSE' and (bpm is null or musical_key is null);
+  update tracks set bpm = 135, musical_key = 'B minor' where audio_upload_id = '01JCP6YS3VED0G57BV51Q8C9FV' and (bpm is null or musical_key is null);
+  update tracks set bpm = 161.9, musical_key = 'D major' where audio_upload_id = '01JCP775XWH18JP6YMVW1PDVQ5' and (bpm is null or musical_key is null);
+  update tracks set bpm = 82.5, musical_key = 'D major' where audio_upload_id = '01JCP703YRFB0HCE0ZMJDX47ZR' and (bpm is null or musical_key is null);
+  update tracks set bpm = 172.3, musical_key = 'G major' where audio_upload_id = '01JCP6VXJA958Y4S8KCH61R3GQ' and (bpm is null or musical_key is null);
+  update tracks set bpm = 86, musical_key = 'A minor' where audio_upload_id = '01JCP73RAMM2G313G6Y2T9GB7A' and (bpm is null or musical_key is null);
+  update tracks set bpm = 87, musical_key = 'D major' where audio_upload_id = '01JCP76R8FDGM4601EBHEMWHZY' and (bpm is null or musical_key is null);
+  update tracks set bpm = 79.9, musical_key = 'E minor' where audio_upload_id = '01JCP73EY3C1N7NXGTTPXVVMT8' and (bpm is null or musical_key is null);
+  update tracks set bpm = 87.7, musical_key = 'G major' where audio_upload_id = '01JCP74QH9H25WY5R1SC1ABQ3K' and (bpm is null or musical_key is null);
+  update tracks set bpm = 77.8, musical_key = 'C major' where audio_upload_id = '01JCP77HAEZMCP6Y9KJHNJNB2C' and (bpm is null or musical_key is null);
+  update tracks set bpm = 83.6, musical_key = 'B major' where audio_upload_id = '01JCP742YG3Y4PFEDZFY48978Q' and (bpm is null or musical_key is null);
+  update tracks set bpm = 75.8, musical_key = 'B minor' where audio_upload_id = '01JCP6XGD76A7PRAKJZ045ZTWR' and (bpm is null or musical_key is null);
+  update tracks set bpm = 128, musical_key = 'F minor' where audio_upload_id = '01JCP93BXCCVFB2X0WQRFB7VA6' and (bpm is null or musical_key is null);
+  update tracks set bpm = 144, musical_key = 'A major' where audio_upload_id = '01JCPB3S60SRBEEP76GACJR95W' and (bpm is null or musical_key is null);
+  update tracks set bpm = 108.1, musical_key = 'G flat minor' where audio_upload_id = '01JCP80NWF6H5KV4DTNXT2JGH5' and (bpm is null or musical_key is null);
+  update tracks set bpm = 87.3, musical_key = 'B minor' where audio_upload_id = '01JCQW880V32KB876J6975YM7G' and (bpm is null or musical_key is null);
+  update tracks set bpm = 100, musical_key = 'F major' where audio_upload_id = '01JCP83D5HG0R7D2PAZTDH74R6' and (bpm is null or musical_key is null);
+  update tracks set bpm = 104, musical_key = 'C minor' where audio_upload_id = '01JCP86TGWYAKQ54E8Z4NW8TMK' and (bpm is null or musical_key is null);
+  update tracks set bpm = 154.4, musical_key = 'D minor' where audio_upload_id = '01JCP9EJQS03NQWJ7H0RDD74AR' and (bpm is null or musical_key is null);
```

Comment:
you can see here that this track is now going to have key and bpm set, this one was flagged as having missing fields
https://audius.co/6eve9/soul-render
https://audius-metadata-3.figment.io/v1/full/tracks?permalink=%2F6eve9%2Fsoul-render&user_id=1PqKz&api_key=8acf5eb7436ea403ee536a7334faa5e9ada4b50f&app_name=audius-client